addons/vehicles/CfgVehicles.hpp

```diff
@@ -1,6 +1,7 @@
 class CfgVehicles {
     class B_Heli_Light_01_F;
     class CLASS(Heli_I_MH6_Blue): B_Heli_Light_01_F {
+        MACRO_SET_VEHICLE_CARGO
```

Comment:
What does this contain that doesn't fit a PMC?

Reply:
It comes with some smokes, MXC rifles and ammo and steerable parachutes. So not much "issue" at all compared to something like the Eagles and the german equipment they add.